src/chatops/custom-runner/fly.md

```diff
@@ -0,0 +1,64 @@
+---
+uid: runner-on-fly
+Title: "Tutorial: Deploying a runner to Fly.io"
+---
+
+## Tutorial: Deploying a runner to Fly.io
+
+[Fly.io](https://fly.io) is a platform for quickly launching applications all over the world.
+It's a great platform for hosting Abbot skill runners, since you can quickly launch any Docker image you want.
+
+To begin, make sure you've signed up for an account at [Fly.io](https://fly.io).
+Download the [Fly CLI](https://fly.io/docs/hands-on/install-flyctl/) and log in to it.
+
+Create a new Fly application by moving to an empty folder on your machine and running the following command:
+
+```
+> flyctl launch
+```
+
+Enter any app name you'd like, select the organization and region (we recommend `sea`).
+You **do not** need to create a Postgresql or Redis database to deploy the runner.
+
+Create a `Dockerfile` with the following text:
+
+```dockerfile
+FROM abbotpublic.azurecr.io/runners/python:latest
+ENV ABBOT_SANDBOX_POLICY=none
+```
+
+Generate a random token to use to authenticate Abbot and the runner.
+This token ensures that only your organization can connect to and run skills on the runner.
+To generate the token, just generate a random string, at least 64 characters long, and save it as a secret in Fly:
+
+```shell
+> ABBOT_SKILL_RUNNER_TOKEN=$(openssl rand -hex 64)
+> flyctl secrets set ABBOT_SKILL_RUNNER_TOKEN="$ABBOT_SKILL_RUNNER_TOKEN"
+```
+
+> [!NOTE]
+> Don't lose ABBOT_SKILL_RUNNER_TOKEN! If you do, you'll need to regenerate it and redeploy.
+
+Now, you can deploy the app:
+
+```shell
+> flyctl deploy
+```
+
+Validate that the runner is working by accessing the `https://your-app-name.fly.dev/api/v1/status` endpoint (replacing `your-app-name` with whatever app name you chose when creating the Fly app).
+You should see a `"status": "ok"` field in the JSON payload.
+
+Now, go to your [Organization Settings](https://app.ab.bot/settings/organization) in Abbot, and select "Custom Runners" on the far right.
+If you don't see "Custom Runners", contact `support@ab.bot` to ask about enabling this feature for your account.
```

Comment:
I think we should after these docs are published. We can update that page to link to the docs.

Reply:
Yeah, I suppose we can turn it on for everyone. It'll need a little bit of work though, I need to clean up some of the old code. Let's do that in a follow up PR to abbot. I'll remove this warning from the docs though, there's a risk someone comes here and wants to know why they don't see the option, but we can handle that.

Reply:
It's definitely not a high-priority thing. I was asking in the interest of reducing friction for people since it doesn't realllly matter to us if it's on or off.

Reply:
It's very little churn, mostly deleting the code and UI that showed Docker image URLs and passwords for the old custom runner process.